assets/models/system/automl-image-classification/description.md

```diff
@@ -0,0 +1,80 @@
+
+### Finetune samples
+
+Task|Dataset|Python sample (Notebook)|CLI with YAML
+|---|--|--|--|
+Image Multi-class classification|[fridgeObjects](https://cvbp-secondary.z19.web.core.windows.net/datasets/image_classification/fridgeObjects.zip)|<a href="https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb" target="_blank">image-classification-multiclass.ipynb</a>|<a href="https://github.com/Azure/azureml-examples/blob/main/cli/jobs/automl-standalone-jobs/cli-automl-image-classification-multiclass-task-fridge-items/cli-automl-image-classification-multiclass-task-fridge-items.yml" target="_blank">image-classification-multiclass.yml</a>
+Image Multi-label classification|[multilabel fridgeObjects](https://cvbp-secondary.z19.web.core.windows.net/datasets/image_classification/multilabelFridgeObjects.zip)|<a href="https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb" target="_blank">image-classification-multilabel.ipynb</a>|<a href="https://github.com/Azure/azureml-examples/blob/main/cli/jobs/automl-standalone-jobs/cli-automl-image-classification-multilabel-task-fridge-items/cli-automl-image-classification-multilabel-task-fridge-items.yml" target="_blank">image-classification-multilabel.yml</a>
+
+
+### Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-sdk-image-classification" target="_blank">image-classification-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-cli-image-classification" target="_blank">image-classification-online-endpoint.sh</a>
+Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-image-classification" target="_blank">image-classification-batch-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-batch-cli-image-classification" target="_blank">image-classification-batch-endpoint.sh</a>
+
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input
+
+```json
+{
+   "input_data":{
+      "columns":[
+         "image"
+      ],
+      "index":[0, 1],
+      "data":[
+         ["image1"],
+         ["image2"]
+      ]
+   }
+}
+```
+Note:
+- "image1" and "image2" should be strings in `base64` format.
+
+#### Sample output
+
+```json
+[
+    {
+        "probs": [0.95, 0.03],
+        "labels": ["label1", "label2"]
+    },
+    {
+        "probs": [0.04, 0.93],
+        "labels": ["label1", "label2"]
+    }
+]
+```
+
+#### Model inference - visualization
+For a sample image below, the top 4 labels are 'trilobite', 'affenpinscher, monkey pinscher, monkey dog', 'mosque', 'Bernese mountain dog'.
```

Comment:
the highest one doesn't seem to match with the image ...

Reply:
Actually I'm not sure that any of these labels actually match the image (even the dog specie).

Reply:
yep, that is why I wanted to use the fine-tuned images (as discussed yesterday). Results are not very good on the images from https://brandcentral.microsoft.com/. Please let me know if we are allowed to use any other images. I am only aware of brandcentral.